charts/prometheus-adapter/values.yaml

```diff
@@ -151,6 +151,14 @@ tls:
   certificate: |-
     # Public key of the APIService
 
+# Set environment variables from given secrets or configmaps
+envFrom: []
```

Comment:
can we just use "env" please.
it's mor flexible and can refetence configmaps and secrets too.
https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#define-container-environment-variables-using-configmap-data
https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-environment-variables

Reply:
Yes, changed.